opentech/apply/funds/views.py

```diff
@@ -80,19 +80,24 @@ class SubmissionListView(AllActivityContextMixin, BaseAdminSubmissionsTable):
     template_name = 'funds/submissions.html'
 
     def get_context_data(self, **kwargs):
-        base_query = RoundsAndLabs.objects.with_progress().active().order_by('-end_date')
-        open_rounds = base_query.open()[:6]
-        open_query = '?round_state=open&active=active'
-        closed_rounds = base_query.closed()[:6]
-        closed_query = '?round_state=closed&active=active'
-
-        return super().get_context_data(
-            open_rounds=open_rounds,
-            open_query=open_query,
-            closed_rounds=closed_rounds,
-            closed_query=closed_query,
-            **kwargs,
-        )
+        # For OTF staff, display rounds/labs where user is the lead
+        if self.request.user.is_staff:
```

Comment:
This is for the dashboard - please update the view here: https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/apply/dashboard/views.py#L11